cobigen-templates/pom.xml

```diff
@@ -27,12 +50,9 @@
 
   <build>
     <resources>
-      <resource>
-        <directory>src/main/templates</directory>
-        <targetPath>src/main/templates</targetPath>
-      </resource>
```

Comment:
Taking this out doesn't hurt backwards compatibility?

Reply:
It's still there, just moved to line 52.
I've just changed our IDE path to src/main/resources.
The path in the jar remains src/main/templates.